pipelines/tasks/hisat2.wdl

```diff
@@ -53,15 +56,19 @@ task HISAT2rsem {
       --new-summary --summary-file ${output_name}.log \
       --met-file ${output_name}.hisat2.met.txt --met 5 \
       -k 10 \
+      --mp 1,1 \
```

Comment:
Same comment here; please explain parameter choices. Ideally I should be able to pick up your task wdls and understand how to use them and what they do (beyond just "aligns reads").